### PR TITLE
A heck ton of flags. State_62_LoadedSaveSpawn_45ADD0().

### PR DESCRIPTION
**Pull request by d3nwah**

### CODE REVIEW

File: Source/AliveLibAO/Abe.cpp [hunk 1]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+};
+
+void Abe::State_62_LoadedSaveSpawn_REAL_45ADD0()
 {
     NOT_IMPLEMENTED();
 }
 
+void Abe::State_62_LoadedSaveSpawn_45ADD0()
+{
+    Event_Broadcast_417220(6, this);
```
Comment (mouzedrift) [inline Source/AliveLibAO/Abe.cpp:7004]: enum

File: Source/AliveLibAO/Abe.cpp [hunk 2]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+    LevelIds field_29A_continue_level;
+    FP field_29C_elum_sprite_scale;
+    int pad[7];
+    char field_0;
+};
+
+using elumSpawn = decltype(&Elum::Spawn_410E90);
+const elumSpawn dword_4CF550[14] =
+{
+    NULL,
```
Comment (paulsapps) [inline Source/AliveLibAO/Abe.cpp:6981]: NULL's -> nullptrs

File: Source/AliveLibAO/Abe.cpp [hunk 3]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+    WORD field_296_elum_zone_number;
+    WORD field_298_elum_continue_path;
+    LevelIds field_29A_continue_level;
+    FP field_29C_elum_sprite_scale;
+    int pad[7];
+    char field_0;
+};
+
+using elumSpawn = decltype(&Elum::Spawn_410E90);
+const elumSpawn dword_4CF550[14] =
```
Comment (paulsapps) [inline Source/AliveLibAO/Abe.cpp:6979]: [14] -> [] to infer the size

File: Source/AliveLibAO/Abe.cpp [hunk 4]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+            sActiveHero_507678->field_AC_ypos + FP_FromInteger(60),
+            &pLine2,
+            &hitX2,
+            &hitY2,
+            1 << pSaveData->field_23A_mode_mask
+            ))
+        {
+            sActiveHero_507678->field_F4_pLine = pLine2;
+            sActiveHero_507678->field_AC_ypos = hitY2;
+            sActiveHero_507678->field_FC_current_motion = 0;
```
Comment (paulsapps) [inline Source/AliveLibAO/Abe.cpp:7027]: missing enum on most instances of field_FC_current_motion  = x

File: Source/AliveLibAO/Abe.cpp [hunk 5]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+                    else
+                    {
+                        gElum_507680->field_FC_current_motion = eElumStates::State_21_Land_414A20;
+                        gElum_507680->field_E4_previous_motion = eElumStates::State_21_Land_414A20;
+                    }
+                }
+            }
+            else
+            {
+                gElum_507680->field_F4_pLine = (PathLine *) -2;
```
Comment (paulsapps) [inline Source/AliveLibAO/Abe.cpp:7140]: reinterpret_cast, this is looking like some OWI hack as well, setting a pointer to -2 is very suspect

File: Source/AliveLibAO/Abe.cpp [hunk 6]
```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
...
+            }
+            if (gElum_507680->field_170_flags.Get(Elum::Flags_170::eFoundHoney_Bit4))
+            {
+                if (!ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 203, 0, 0))
+                {
+                    ResourceManager::LoadResourceFile_455270("ELMHONEY.BAN", 0);
+                }
+                gElum_507680 = gElum_507680;
```
Comment (paulsapps) [inline Source/AliveLibAO/Abe.cpp:7148]: :)

Comment (d3nwah) [reply Source/AliveLibAO/Abe.cpp:7148]: ...sometimes it's important to call a spade a spade :tape:

**Pull request merged**